modin/pandas/test/test_api.py

```diff
@@ -235,6 +235,20 @@ def test_dataframe_api_equality():
     assert not len(difference), "Extra params found in API: {}".format(difference)
 
 
+def test_series_str_api_equality():
+    modin_dir = [obj for obj in dir(pd.Series().str) if obj[0] != "_"]
```

Comment:
In Modin, `str` accessor is implemented differently, so we have to access it through a class instance.